docs/help/diffusion.md

```diff
@@ -0,0 +1,18 @@
+Taking the time to understand the diffusion process will help you to understand how to more effectively use InvokeAI.
+
+There are two main spaces Stable Diffusion works in: image space and latent space.
+
+Image space represents images in pixel form that you look at. Latent space represents compressed inputs. It’s in latent space that Stable Diffusion processes images. A VAE (Variational Auto Encoder) is responsible for compressing and encoding inputs into latent space, as well as decoding outputs back into image space.
+
```

Comment:
Somewhere in the intro you need to explain that the unet model was trained on a large series of latent images to which known amounts of random noise were added. The unet model’s task is to predict that noise. This means that the unet can be given a slightly noisy image and it will predict the pattern of noise needed to subtract from the image in order to recover the original. The text encoder, in turn, is trained to associate patterns of words in a prompt with images; its task is to say, given an image and a prompt, how closely the image matches the prompt.

By working together at each denoising step, the unet denoises the image in such a way that the result is similar to images in the unet’s training set while the text encoder nudges the unet towards creating images that maximize similarity to the prompt.

Reply:
Added a section talking about U-Net & CLIP and their interaction